dev-libs/stfl/files/stfl-0.24-ncurses-widechar.patch

```diff
@@ -0,0 +1,23 @@
+ncurses widechar functions are only available if the correct macros are
+defined. Append CFLAGS from ncursesw pkgconfig to ensure that widechar
+functions are available.
+
+From 10eca6b7eb205ca3b748f10350f4008287e9cd3c Mon Sep 17 00:00:00 2001
+From: Violet Purcell <vimproved@inventati.org>
+Date: Tue, 27 Jun 2023 20:10:27 +0000
+Subject: [PATCH] Define correct macros for ncurses widechar support
+
+--- a/Makefile
++++ b/Makefile
+@@ -21,7 +21,7 @@
+ include Makefile.cfg
+ 
+ export CC = gcc -pthread
+-export CFLAGS += -I. -Wall -Os -ggdb -D_GNU_SOURCE -fPIC
++export CFLAGS += -I. -Wall -Os -ggdb -D_GNU_SOURCE -fPIC $(shell pkg-config --cflags ncursesw)
+ export LDLIBS += -lncursesw
```

Comment:
```suggestion
 export LDLIBS += $(pkg-config --libs ncursesw)
```

... but please replace `pkg-config` with ${PKG_CONFIG}, and add a line doing e.g. `PKG_CONFIG ?= pkg-config` above.

Reply:
Done.  Sorry for being somewhat dense 😅